LICENSE

```diff
@@ -188,6 +188,8 @@
       identification within third-party archives.
 
    Copyright 2019-2023 Andrei Krasnitski
+   Copyright of Loki, Promtail 2023-Present Grafana [AGPL-3.0 license](https://github.com/grafana/loki?tab=AGPL-3.0-1-ov-file#readme)
```

Comment:
If you are not changing anything in Loki, Promtail and Grafana you don't need to add this.

If you want to mention them you can add them somewhere in the readme but not in the licence file. Here is a nice example from the [Go Buffalo Web Framework](https://github.com/gobuffalo/buffalo#shoulders-of-giants).

Reply:
done